javascript/ql/src/semmle/javascript/dataflow/CapturedNodes.qll

```diff
@@ -54,11 +54,11 @@ private predicate exposedAsReceiver(DataFlow::SourceNode n) {
 }
 
 /**
- * A source for which the flow is entirely captured by the dataflow library.
+ * A source that is entirely local, i.e. the dataflow library models all of its flow.
```

Comment:
We generally don't use "i.e." (or "e.g.") in doc comments; suggest replacing with "in the sense that".